_drafts/technical-workflow.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: post
+title:  "The technical workflow of Etsy France"
+date:   2015-02-23 13:55:00
+categories: workflow
+tags: [workflow, jira, github, pr, review]
+author: antoine
+---
+
+Here at Etsy France we tried several organisations.  
```

Comment:
several **kinds of organization** (et un z dans toutes les occurrences du mot "organization")